sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Feature Usage Events
+description: |-
+  Metrics from events pings for mobile feature usage dashboards (iOS)
+owners:
+- rzhao@mozilla.com
+labels:
```

Comment:
We should consider adding `application` label to all of the new `metadata.yaml` files being added via this PR as per [recommended_practices](https://mozilla.github.io/bigquery-etl/reference/recommended_practices/) docs page under the `Query Metadata` section.

Example: 
https://github.com/mozilla/bigquery-etl/blob/dffbb3afef5a3b0b5adc396fa4b874bc0f617b6b/sql/moz-fx-data-shared-prod/fenix_derived/events_daily_v1/metadata.yaml#L11